src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -200,4 +207,24 @@ public function isRaw()
     {
         return $this->raw;
     }
+
+    /**
+     * Gets the SameSite attribute.
+     *
+     * @return string|bool
+     */
+    public function getSameSite()
+    {
+        return $this->sameSite;
+    }
+
+    /**
+     * Checks if the cookie value should be sent with a SameSite attribute.
+     *
+     * @return bool
+     */
+    public function hasSameSite()
```

Comment:
I suggest to remove this method, using `getSameSite` is the way to go, no need for such a simple "helper"